src/label_studio_sdk/label_interface/interface.py

```diff
@@ -839,9 +840,66 @@ def generate_sample_task(self, mode="upload", secure_mode=False):
 
         return task
 
-    def generate_sample_annotation(self):
-        """ """
-        raise NotImplemented()
+    def generate_sample_prediction(self) -> Optional[dict]:
+        """Generates a sample prediction that is valid for this label config.
+
+        Example:
+            {'model_version': 'sample model version',
+             'score': 0.0,
+             'result': [{'id': 'e7bd76e6-4e88-4eb3-b433-55e03661bf5d',
+               'from_name': 'sentiment',
+               'to_name': 'text',
+               'type': 'choices',
+               'value': {'choices': ['Neutral']}}]}
+
+        NOTE: `id` field in result is not required when importing predictions; it will be generated automatically.
+        NOTE: for each control tag, depends on tag.to_json_schema() being implemented correctly
+        """
+        prediction = PredictionValue(
+            model_version='sample model version',
+            result=self.create_regions({
```

Comment:
result generation can be moved to a separate function to avoid double editing in the future